content/en/community/end-user/interviews-feedback.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Interviews or Feedback Sessions
+Description: >-
+ Help improve OpenTelemetry - share your feedback directly with us!
+---
+
+One of the core functions of the OpenTelemetry End User Working Group is 
+to improve the project by gathering feedback from end users and sharing
+them with the appropriate SIGs to help drive prioritization of 
+improvements and changes to the project. 
+
+Direct interview or feedback sessions between an organization and the 
+OpenTelemetry Community is one such resource. In these hour-long sessions, 
+we learn about the organization's OTel adoption and implementation, with 
+particular interest in the following:
+
+* Special use cases 
+* Challenges with implementation -- what went well? What needs improvement? 
+  These can be anything, from instrumentation to documentation. 
+* Challenges with adoption within the organization -- how can the 
+  Community help you drive adoption with your teams? 
+
+We will then take the feedback you shared and turn them into actionable tasks 
+for the relevant SIGs, as well as provide context and insight for you during
+the session as appropriate. 
+
+We schedule these sessions once a month, usually during one of the EUWG's
+meetings (alternating Thursdays at 10AM Pacific Time), but can set up a 
+different time if that does not work for you. The sessions are typically 
+attended by a few project maintainers and/or Governance Committee members, 
+as well as some curious general members of the Community. 
+
+Things to keep in mind:
+* These sessions are recorded. However, if your organization has compliance
+  concerns, we can delete them once we've summarized the notes and feedback. 
+* We will write up a summary post for the OpenTelemetry blog. Our goal with
+  these is to drive awareness and adoption of the software, and to chronicle
+  interesting use cases. These can be anonymized if your org has compliance
+  concerns. 
+* These are open to the public, so other users may come and ask you questions.
+  
+To schedule a session, reach out to us in [CNCF Community Slack](https://communityinviter.com/apps/cloud-native/cncf):
+* In the [#otel-user-research](https://cloud-native.slack.com/archives/C01RT3MSWGZ) channel
+* Via direct message -- ping Reese Lee, Adriana Villela, or Rynn Mancuso 
```

Comment:
```suggestion
* Via direct message: ping Reese Lee, Adriana Villela, or Rynn Mancuso 
```

Reply:
you can also link to the people directly in CNCF Slack.